metecho/api/serializers.py

```diff
@@ -585,13 +594,15 @@ class TaskSerializer(HashIdModelSerializer):
         required=False,
         allow_null=True,
     )
-    project = serializers.PrimaryKeyRelatedField(
+    project = NestedPrimaryKeyRelatedField(
         queryset=Project.objects.all(),
         pk_field=serializers.CharField(),
+        serializer=ProjectMinimalSerializer,
         required=False,
         allow_null=True,
     )
     root_project = serializers.SerializerMethodField()
+    root_project_slug = serializers.SerializerMethodField()
```

Comment:
```suggestion
    root_project_slug = serializers.CharField(source="root_project.slug", read_only=True)
```
I prefer this style because it's more clear and compact than adding the extra method